src/errors/test-run/index.js

```diff
@@ -70,14 +70,24 @@ export class DomNodeClientFunctionResultError extends TestRunErrorBase {
     }
 }
 
+// Selector errors
```

Comment:
Why we need to two groups of `Selector errors`? https://github.com/DevExpress/testcafe/pull/603/files#diff-ab5af4bc525a2e3c00245caa5503456eR219